### PR TITLE
ROC-5473-claim-submission-operation-indicator

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5473

### Change description ###
Adding claim submission operation indicator and prepare for resilience.

### CODE REVIEW

File: src/main/resources/db/migration/claimstore/V2019_04_12_1414__schema_change_for_claim_submission_operation_indicator.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE claim
+  ADD COLUMN submission_operation_ind JSONB NULL;
```
Comment (sabahirfan) [inline src/main/resources/db/migration/claimstore/V2019_04_12_1414__schema_change_for_claim_submission_operation_indicator.sql]: How about `submission_operation_indicators`?

File: src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java [hunk 2]
```diff
@@ -233,4 +233,13 @@ Integer linkDefendant(
         @Bind("defendantId") String defendantId,
         @Bind("defendantEmail") String defendantEmail
     );
+
+    @SqlUpdate(
+            "UPDATE claim SET claim_documents = :claimDocumentCollection::JSONB"
+            + " WHERE id = :claimId"
+    )
+    Integer saveClaimDocuments(
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java]: This method should be updating operation indicators not the claim documents. This looks duplicate of:
https://github.com/hmcts/cmc-claim-store/blob/9024dad6a1c35b240077713377d74af627789108/src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java#L152

Comment (karthikkannankainos) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java]: @sabahirfan this is partial commit. I was working when I have to commit.

File: src/main/resources/db/migration/claimstore/V2019_04_12_1414__schema_change_for_claim_submission_operation_indicator.sql (hunk 1 above)
Comment (karthikkannankainos) [reply src/main/resources/db/migration/claimstore/V2019_04_12_1414__schema_change_for_claim_submission_operation_indicator.sql]: @sabahirfan sure

Comment (karthikkannankainos) [reply src/main/resources/db/migration/claimstore/V2019_04_12_1414__schema_change_for_claim_submission_operation_indicator.sql]: Checked. The latest Postgress allows 63 characters. So will change it

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDClaimSubmissionOperationIndicators.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class CCDClaimSubmissionOperationIndicators {
+    private CCDYesNoOption claimantNotification;
+    private CCDYesNoOption defendantNotification;
+    private CCDYesNoOption bulkPrint;
+    private CCDYesNoOption RPA;
```
Comment (sabahirfan) [inline ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDClaimSubmissionOperationIndicators.java]: Should we rename this to `roboticProcessAutomation`?

**Comment by karthikkannankainos** [comment]
Conflict in git merge. Due to way git merge is done, cant proceed, so create a new branch

**Comment by karthikkannankainos** [comment]
Conflict in git merge. Due to way git merge is done, cant proceed, so create a new branch

**Pull request closed without merging**